src/client/index.js

```diff
@@ -38,7 +38,10 @@ class Hammerhead {
         // Methods
         this.getOriginElementAttributes = CodeInstrumentation.getAttributesProperty;
         this.upload                     = this.sandbox.upload.upload;
-        this.get                        = require;
+
+        // We should provide a function to retrieve modules,
+        // since the client tests do not support the ECMAScript 6 modules syntax
+        this.get = require;
```

Comment:
It's not because it doesn't support, but because hammerhead will be bundled into the single file and we will not have access to the internal modules by default